packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -222,59 +223,71 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
     <div className={styles.formContainer}>
       <Stack gap={7}>
         <FormGroup legendText={t('condition', 'Condition')}>
-          <Search
-            size="md"
-            id="conditionsSearch"
-            labelText={t('enterCondition', 'Enter condition')}
-            light={isTablet}
-            placeholder={t('searchConditions', 'Search conditions')}
-            onChange={handleSearchTermChange}
-            onClear={() => setSelectedCondition(null)}
-            disabled={context === 'editing'}
-            value={(() => {
-              if (conditionToLookup) {
-                return conditionToLookup;
-              }
-              if (selectedCondition) {
-                return selectedCondition.display;
-              }
-              return '';
-            })()}
-          />
-          <div>
-            {(() => {
-              if (!conditionToLookup || selectedCondition) return null;
-              if (isSearchingConditions)
-                return <InlineLoading className={styles.loader} description={t('searching', 'Searching') + '...'} />;
-              if (conditions && conditions.length) {
-                return (
-                  <ul className={styles.conditionsList}>
-                    {conditions?.map((condition, index) => (
-                      <li
-                        role="menuitem"
-                        className={styles.condition}
-                        key={index}
-                        onClick={() => handleConditionChange(condition)}
-                      >
-                        {condition.display}
-                      </li>
-                    ))}
-                  </ul>
-                );
-              }
-              return (
-                <>
-                  <Layer>
-                    <Tile className={styles.emptyResults}>
-                      <span>
-                        {t('noResultsFor', 'No results for')} <strong>"{conditionToLookup}"</strong>
-                      </span>
-                    </Tile>
-                  </Layer>
-                </>
-              );
-            })()}
-          </div>
+          {context === 'editing' ? (
```

Comment:
When editing a condition, we should show the `display` value property from the `condition` prop. This value is immediately available, unlike the `selectedCondition` property. That means we shouldn't need to show a loading state either. This should be refactored to:

```tsx
const displayName = getFieldValue(condition?.cells, 'display');
```

Which could then get used as follows:

```tsx
{context === 'editing' ? (
  <FormLabel className={styles.conditionLabel}>. 
    {displayName}
  </FormLabel>
...
}
```

Reply:
oh okay, I see that, thanks.